docs/best_practices_guide.adoc

```diff
@@ -1,6 +1,6 @@
 // vim: ft=asciidoc
 
-= openshift-ansible Best Practices Guide
+= Ansible Best Practices Guide
```

Comment:
Also ambiguous as this refers to best practices for this repo and not ansible as a whole